apps/src/codebridge/Workspace/HeaderButtons.tsx

```diff
@@ -77,8 +109,40 @@ const WorkspaceHeaderButtons: React.FunctionComponent = () => {
     sendPythonCodeToMicroBit(pythonCode);
   };
 
+  const onClickSettings = async () => {
+    const results = await dialogControl?.showDialog({
+      type: DialogType.GenericDropdown,
+      title: 'Settings',
+      message: 'Customize your text editor font size',
+      selectedValue: selectedFontSizeKey,
+      items: fontSizeOptions,
+      dropdownLabel: '',
+    });
+    const selectedKey = extractUserInput(results) as keyof typeof FontSize;
+    if (selectedKey && FontSize[selectedKey]) {
+      // We want the user preference for selected font size to persist across a session
+      // for signed-in users per app type.
+      if (signInState === SignInState.SignedIn) {
```

Comment:
We will want this setting to persist across sessions. Up to you if you want to leave this in and do that as a follow-up or rework it now. We can use [UserPreferences](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/lib/util/UserPreferences.js) to persist a setting.

Reply:
Thanks! I'll handle in a follow-up.